python/Ganga/testlib/functions.py

```diff
@@ -0,0 +1,18 @@
+
+import random
+import tempfile
+import string
+
+def generateUniqueTempFile(ext = '.txt'):
```

Comment:
This is yet another reason why I didn't want to put these common 4/5 lines in a global function. I'll move them back to the tests if this starts to become a costly exercise.